microservices/nris_api/backend/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.6
+
+ENV LD_LIBRARY_PATH="/opt/oracle/instantclient" \
+    OCI_HOME="/opt/oracle/instantclient" \
+    OCI_LIB_DIR="/opt/oracle/instantclient" \
+    OCI_INCLUDE_DIR="/opt/oracle/instantclient/sdk/include"
+
+COPY /oracle/ .
+
+# Update installation utility
+RUN apt-get update
+
+RUN BUILD_PACKAGES="build-essential unzip curl libaio1 python-minimal git" && \
+    apt-get install -y $BUILD_PACKAGES
+
+RUN mkdir -p opt/oracle && \
+    unzip instantclient-basic-linux.x64-19.3.0.0.0.zip -d /opt/oracle && \
+    unzip instantclient-sdk-linux.x64-19.3.0.0.0.zip -d /opt/oracle && \
+    mv /opt/oracle/instantclient_19_3 /opt/oracle/instantclient && \
+    echo '/opt/oracle/instantclient/' | tee -a /etc/ld.so.conf.d/oracle_instant_client.conf && ldconfig && \
+    rm -rf instantclient-basic-linux.x64-19.3.0.0.0.zip instantclient-sdk-linux.x64-19.3.0.0.0.zip && \
+    rm -rf /var/lib/apt/lists/*
+
+# Create working directory
+RUN mkdir /app
+WORKDIR /app
+
+# Install the requirements
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Run the server
+EXPOSE 5500
+CMD [ "flask", "run"]
```

Comment:
You execute `flask run` here and in docker-compose? As far as I can tell, that will cause it to run twice. Is that intentional?

Reply:
Same as above, i'll take a look .

Reply:
Passing 'CMD' in docker-compose overwrites the CMD in the dockerfile, not an addendum. Only one CMD per container.